ipynb/AG_stool_temporal dynamics.ipynb

```diff
@@ -0,0 +1,664 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "import skbio\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "import seaborn"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "dm = skbio.DistanceMatrix.read('ag-cleaned-multiple-timepoints_even1250_unweighted.txt')\n",
+    "w_dm = skbio.DistanceMatrix.read('ag-cleaned-multiple-timepoints_even1250_weighted.txt')\n",
+    "deltas = pd.read_csv('ag-cleaned-multiple-timepoints-deltas.txt', sep='\\t', dtype=str)"
```

Comment:
Can you provide a description of what this file is? And how it was created?

Reply:
How's this?

"Using the full American Gut mapping file, we reduced it to only samples that were fecal. The mapping file was then filtered to remove samples which did not have sufficient data (after bloom removal). Following this, host subject IDs representing only a single sample were removed. The time deltas were then generated by iterating over each host subject ID, and computing the time difference in days between all samples associated with that host subject ID."